batch/batch.go

```diff
@@ -23,8 +23,11 @@ import (
 	"time"
 )
 
+// DefaultWaitInterval is the default WaitInterval for Func.
+var DefaultWaitInterval = 500 * time.Microsecond
+
 // DefaultMaxDuration is the default MaxDuration for Func.
-var DefaultMaxDuration = 200 * time.Microsecond
+var DefaultMaxDuration = 10 * time.Millisecond
```

Comment:
It'd be nice to have numbers to back this default. If we could get some traces that roughly how long is reasonable, or if we can track the default "actual wait" times batches. As a lower bound, I've definitely seen failed-to-batch patterns with our current timing.